releasenotes/notes/remove-deprecation-deprecate-sigmoid-319213068ae00ea9.yaml

```diff
@@ -0,0 +1,7 @@
+---
+deprecations:
+  - |
+    Deprecated class ``CrossEntropySigmoidLoss`` as it is not used in the package.
```

Comment:
I am trying to understand this statement. The class exists as a loss function and could be specified on TrainableModel as the loss to be used, so `it is not used in the package` is hard for me to follow, as it seems it could be used and specified as a loss where its applicable for an algorithm.

Reply:
OK, may be unclear. The loss function is not called anywhere in QML. It could be used, but never used actually. We don't actually need it.

Reply:
That's right. It can be used externally, but you wrote "is not useful to keep". That's the only reason I want to deprecate it.